bot/exts/info/doc/_cog.py

```diff
@@ -34,6 +35,7 @@
     "pdbcommand",
     "2to3fixer",
 )
+DELETE_ERROR_MESSAGE_REACTION = '\u274c'  # :x:
```

Comment:
You can actually refer to Unicode characters by their name, that might be a better option.

Reply:
See the first option..
```py
>>> "\N{GREEK CAPITAL LETTER DELTA}"  # Using the character name
'\u0394'
>>> "\u0394"                          # Using a 16-bit hex value
'\u0394'
>>> "\U00000394"                      # Using a 32-bit hex value
'\u0394'
```
From Python 3 docs - [Unicode HOWTO](https://docs.python.org/3/howto/unicode.html).